resolwe/flow/views/data.py

```diff
@@ -165,6 +165,11 @@ def perform_get_or_create(self, request, *args, **kwargs):
 
     def perform_create(self, serializer):
         """Create a resource."""
+        process = serializer.validated_data.get('process')
+        if not process.is_active:
+            raise exceptions.ParseError('Process retired (id: {}, slug: {}/{}).'.format(
```

Comment:
New line after opening bracket of `ParseError` and before closing one.

Reply:
done